utils/formValidations.ts

```diff
@@ -159,4 +160,45 @@ export function validateFloat(float: string): boolean {
     }
 
     return true
-}
\ No newline at end of file
+}
+
+export function validateUserSubmittedLastName(name: string): boolean {
+    // The last name cannot be an empty space like " "
```

Comment:
I realized while writing the tests that the user could just hit the space bar and it would pass validation. So, this trims the name to make sure a string with infinite empty spaces can't pass.